docs/web-ui.md

```diff
@@ -369,3 +369,44 @@ The web UI includes a Streaming tab if the application uses Spark streaming. Thi
 scheduling delay and processing time for each micro-batch in the data stream, which can be useful
 for troubleshooting the streaming application.
 
+## JDBC/ODBC Server Tab
+We can see this tab when Spark is running as a [distributed SQL engine](sql-distributed-sql-engine.html) and shows information about sessions and SQL jobs submitted.
```

Comment:
We can see this tab when Spark is running as a [distributed SQL engine](sql-distributed-sql-engine.html). It shows information about sessions and SQL jobs submitted.